languages/swift/exercises/concept/enums/.docs/after.md

```diff
@@ -0,0 +1,284 @@
+The built-in Swift types can each represent a finite number of values. These numbers are usually quite high, e.g. 2<sup>32</sup> different `Float`s 2<sup>64</sup> different `Int`s, ~2<sup>20</sup> different `Character`s, and a number of strings bounded only by the amount of storage on your computer. On the flip side, there are a handful of types that offer just a few values. The `Never` type has no values, `Void` has just `()`, and `Bool` has just `true` and `False`.
```

Comment:
Do students know about `Never`, `Void` and `Bool` at this point? I think you might be able to shorten this explanation a little bit. In fact, if you would just have the next section (with a little rewriting) that might actually be enough.

Reply:
They know about `Void` and `Bool`, but I rewrote that section as it was more than was needed and got rid of those references.

Reply:
Great!